### PR TITLE
populate jobIds in resources for GET /taas-teams/:id

### PR DESCRIPTION
**Pull request by imcaizheng**
and populate sklls for GET /taas-teams/:id/jobs/:jobId

### CODE REVIEW

**Comment by maxceem** [comment]
Looks good. Merging for quick testing in DEV env. 

FYI @nkumar-topcoder

File: src/services/TeamService.js [hunk 1]
```diff
@@ -125,21 +137,6 @@ async function getTeamDetail (currentUser, projects, isSearch = true) {
...
-              item.customerRate = findRbs.customerRate
-              const job = _.find(jobs, { id: findRbs.jobId })
-              if (job) {
-                item.job = {
-                  id: job.id,
-                  name: job.description
-                }
-              }
-            }
-          }
```
Comment (maxceem) [inline src/services/TeamService.js:142]: Just to confirm @imcaizheng before I didn't see this returned for `resources` were there some kind of issue?

Comment (imcaizheng) [reply src/services/TeamService.js:142]: The resources field is attached to res at https://github.com/topcoder-platform/taas-apis/pull/38/files/e192960abae4473304a1b5cd92820d36578c3de5#diff-821570075559f59a1fc7d0ac9e41b013516a19e4d7a2e81c76d3b5168589c2d3R129

Comment (imcaizheng) [reply src/services/TeamService.js:142]: I tested locally. The jobIds in resources are populated correctly.

**Pull request merged**